tech/languages/csharp/mono-installation.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Mono
+page: csharp
+order: 1
+section: tech-languages
+description: C# provided by Mono: a Cross-platform, Open Source, .NET development framework
```

Comment:
I just copy&pasted from https://admin.fedoraproject.org/pkgdb/package/mono/
I will fix it to lower case